spec/menu_spec.rb

```diff
@@ -0,0 +1,76 @@
+
+require 'menu'
+
+describe "Menu" do
+
+  describe "#view_menu" do
+    it "Displays the menu" do
+      menu = Menu.new('lib/menu.yaml')
+      expect(menu.view_menu).to eq (YAML.load_file('lib/menu.yaml'))
+    end
+  end
+
+  describe "#choose" do
+
+    it "allows to pick an item number and stipulate its quantity" do
+      menu = Menu.new('lib/menu.yaml')
+      menu.choose(:"Kielbasa chicken", 2)
+      expect(menu.basket).to eq([{:"Kielbasa chicken"=>4}, {:"Kielbasa chicken"=>4}])
```

Comment:
Metrics/LineLength: Line is too long. [85/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)